accepted/0018-notification-widget.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2020-04-17
+- RFC PR: #18
+- Mantis Issue: N/A
+
+- Related PRs:
+	1. https://github.com/obsproject/obs-studio/pull/1832
+
+# Summary
+
+Add the ability to display notifications to the user.
+
+# Motivation
+
+Currenty, there is no way of showing custom notifications to users.
+
+# Detailed design
+
+## User UX
+
+- Display notifications at top of main window
+- Have three notification types (Info, Error, and Warning)
+- Each type will display a different icon and the widget would be a different color for each
+- Display custom text
+- Add button for different types of actions (open source properties, open settings window, open url)
```

Comment:
Your PR only seems to use this to open a URL instead of taking internal actions; any thoughts on how to both present a URL "button" ("click here to learn more about this issues") and a "fix this" button? Are there use-cases that don't need a button?

Also, should there be a limit on amount of text displayed (does the widget support multiple lines)?